srcpkgs/biboumi/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'biboumi'
+pkgname=biboumi
+version=9.0
+revision=1
+build_style=cmake
+configure_args="-DWITHOUT_SYSTEMD=1 -Wno-dev"
+conf_files="/etc/biboumi/biboumi.cfg"
+make_dirs="/var/lib/biboumi/ 0755 _biboumi _biboumi"
+hostmakedepends="python3-Sphinx python3-sphinx_rtd_theme pkg-config"
+makedepends="expat-devel libticonv-devel sqlite-devel libpqxx-devel libidn-devel udns-devel botan-devel postgresql-libs-devel"
+short_desc="XMPP gateway that connects to IRC servers"
+maintainer="Luca Matei Pintilie <luca@lucamatei.com>"
+license="Zlib"
+homepage="https://biboumi.louiz.org/"
+changelog="https://codeberg.org/poezio/biboumi/src/branch/master/CHANGELOG.rst"
+distfiles="https://codeberg.org/poezio/biboumi/archive/${version}.tar.gz"
+checksum="618776cf3dccb59c84322a921681f4b1f8b144b523fd5e118d2b47fb0f3eb43f"
+
+system_accounts="_biboumi"
```

Comment:
should probably have `_biboumi_homedir="/var/lib/biboumi"` which might remove the need for `make_dirs`

Reply:
Doesn't seem like _homedir creates the folder, so I kept the make_dirs line